pkg/controller/pxcbackup/controller.go

```diff
@@ -311,6 +311,9 @@ func (r *ReconcilePerconaXtraDBClusterBackup) runDeleteBackupFinalizer(ctx conte
 	for _, f := range cr.GetFinalizers() {
 		var err error
 		switch f {
+		case "delete-s3-backup":
+			log.Info("The value delete-s3-backup is deprecated and will be deleted in 1.18.0. Use percona.com/delete-s3-backup")
```

Comment:
Won't `percona.com/delete-s3-backup` be depricated also? percona.com/delete-backup ?

Reply:
we don't have this value before, so I deleted it.